### PR TITLE
Updating changelog

### PR DESCRIPTION
**Pull request by aaronjeline**
## Description of changes
Updates changelog for upcoming 3.2.2
## Issue #, if available
N/A
## Checklist for requesting a review

The change in this PR is (choose one, and delete the other options):

- [X] A change "invisible" to users (e.g., documentation, changes to "internal" crates like `cedar-policy-core`, `cedar-validator`, etc.)

I confirm that [`cedar-spec`](https://github.com/cedar-policy/cedar-spec) (choose one, and delete the other options):

- [X] Does not require updates because my change does not impact the Cedar formal model or DRT infrastructure.

### CODE REVIEW

**Review by khieta — APPROVED** [review]
You'll also want to update the links at the bottom of the file. Probably easiest to wait to merge this until 3.2.2 is released so we can have the correct date.

**Comment by cdisselkoen** [comment]
I think we should name the next patch release 3.2.4, due to the snafu where we had to release 3.2.2 and 3.2.3 as npm-only.  Naming it 3.2.4 will get the crates.io and npm version numbers back in sync.

File: cedar-policy/CHANGELOG.md [hunk 1]
```diff
@@ -82,6 +82,14 @@ Cedar Language Version: 4.0
 - (*) JSON format Cedar policies will now fail to parse if any annotations are not
   valid Cedar identifiers. (#1004, resolving #994)
 
+## [3.2.4] - TBA
+### Changed
+- `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature.
+
+### Fixed
+- JSON format Cedar policies will now fail to parse if any annotations are not valid Cedar identifiers. (#1004, resolving #994)
```
Comment (khieta) [inline cedar-policy/CHANGELOG.md:91]: Also delete these entries from the "Unreleased" section above.

File: cedar-policy/CHANGELOG.md [hunk 2]
```diff
@@ -82,6 +82,14 @@ Cedar Language Version: 4.0
...
 
+## [3.2.4] - TBA
+### Changed
+- `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature.
+
+### Fixed
+- JSON format Cedar policies will now fail to parse if any annotations are not valid Cedar identifiers. (#1004, resolving #994) 
+
 ## [3.2.1] - 2024-05-31
```
Comment (khieta) [inline cedar-policy/CHANGELOG.md:93]: Maybe add a note about why we skipped 3.2.2 & 3.2.3

File: cedar-policy/CHANGELOG.md [hunk 3]
```diff
@@ -82,6 +82,14 @@ Cedar Language Version: 4.0
 - (*) JSON format Cedar policies will now fail to parse if any annotations are not
   valid Cedar identifiers. (#1004, resolving #994)
 
+## [3.2.4] - TBA
+### Changed
+- `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature.
```
Comment (khieta) [inline cedar-policy/CHANGELOG.md:88]: Last comment I swear
```suggestion
- `unknown()` is no longer a valid extension function if `partial-eval`
  is not enabled as a feature. (#1101, resolving #1096)
```

**Comment by khieta** [comment]
Updated in preparation for the upcoming release. Needs re-review.

**Review by shaobo-he-aws — APPROVED** [review]

File: cedar-policy/CHANGELOG.md [hunk 4]
```diff
@@ -79,6 +78,19 @@ Cedar Language Version: 4.0
...
+## [3.2.4] - 2024-08-07
+Cedar Language Version: 3.3
+
+_Note:_ 3.2.2 and 3.2.3 skipped to maintain consistency with the `cedar-wasm` package
+
+### Changed
+
+- (*) `unknown()` is no longer a valid extension function if `partial-eval`
+  is not enabled as a feature. (#1101, resolving #1096)
+
```
Comment (cdisselkoen) [inline cedar-policy/CHANGELOG.md:91]: I'd argue this is `Fixed` rather than `Changed`, I think.  If not, it's technically a breaking change, kinda

**Review by cdisselkoen — APPROVED** [review]

File: cedar-policy/CHANGELOG.md (hunk 4 above)
Comment (khieta) [reply cedar-policy/CHANGELOG.md:100]: Yeah, that's fair. I also think this was a bug fix.

**Review by shaobo-he-aws — APPROVED** [review]

**Pull request merged**